test/stressGUI.cxx

```diff
@@ -2282,9 +2282,9 @@ void testPaletteEditor()
 
 void testHtmlBrowser()
 {
-   TGHtmlBrowser *b = new TGHtmlBrowser("http://bellenot.web.cern.ch/bellenot/Public/html_test/html_test.html");
+   TGHtmlBrowser *b = new TGHtmlBrowser("https://bellenot.web.cern.ch/public/html_test/html_test.html");
```

Comment:
As a side, we should probably move(/copy) those file into the root project area to be a bit more standard.

Reply:
@pcanal You're right, I'll make a more standard test and copy it on the root area. Thanks